tests/scripts/meshcop

```diff
@@ -269,6 +269,7 @@ ba_start()
 
     write_syslog "AGENT: kill old"
     sudo killall "${OTBR_AGENT}" || true
+    sleep 5
```

Comment:
It is actually this line that fixes the CI failure. The problem is that we starts the otbr-agent and immediately after killing it, leaving the TCP socket still running in the background. `SO_REUSEDRR` doesn't always solve this issue because otbr-agent is re-started too quickly before the socket going to the `TIME_WAIT` status.

Sleeping for 5 seconds is for the socket transits to the `TIME_WAIT` status or shutdown completely.